src/components/light.js

```diff
@@ -146,6 +172,50 @@ module.exports.Component = registerComponent('light', {
     this.updateShadow();
   },
 
+  tick: (function tickSetup () {
+    var bbox = new THREE.Box3();
+    var normal = new THREE.Vector3();
+    var cameraWorldPosition = new THREE.Vector3();
+    var tempMat = new THREE.Matrix4();
+    var sphere = new THREE.Sphere();
+    var tempVector = new THREE.Vector3();
+
+    return function tick () {
+      if (
```

Comment:
anyway to simplify this condition and do an early return for readability?

`if (XXX) { return; }`